imports/plugins/core/i18n/client/components/languageDropdown.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
-import { Button, Divider, DropDownMenu, MenuItem } from "/imports/plugins/core/ui/client/components";
+import { Components } from "@reactioncommerce/reaction-components";
```

Comment:
Are we looking at the same thing?  I don't see that.

![register_language_currency_dropdown_components_by_jshimko_ _pull_request__2670_ _reactioncommerce_reaction](https://user-images.githubusercontent.com/3077956/29323305-7e459ce0-81ae-11e7-8960-1954d8327c51.png)